hr_timesheet_activity_begin_end/views/hr_timesheet_sheet.xml

```diff
@@ -7,12 +7,12 @@
       <field name="inherit_id" ref="hr_timesheet_sheet.hr_timesheet_sheet_form"/>
       <field name="arch" type="xml">
         <xpath expr="//field[@name='timesheet_ids']/tree/field[@name='unit_amount']" position="before">
-          <field name="time_start" widget="float_time"/>
-          <field name="time_stop" widget="float_time"/>
+          <field name="time_start" widget="float_time_hms"/>
```

Comment:
@Martronic-SA while testing again the module, we encountered an error : `float_time_hs` should be removed from this file as well